src/commands/dev-env-sync-sql.js

```diff
@@ -35,8 +36,8 @@ function findSiteHomeUrl( sql ) {
 	const results = sql.match( regex );
 
 	if ( results ) {
-		const domain = results[ 2 ].replace( /https?:\/\//, '' );
-		return domain;
+		const url = results[ 2 ];
+		return urlLib.parse( url ).hostname;
```

Comment:
How does this change relate ?

Reply:
`inherit` is the "non-url entry" I was talking about.